packages/blockchain-wallet-v4/src/network/api/swap/types.ts

```diff
@@ -50,9 +50,6 @@ export enum SwapOrderDirection {
   TO_USERKEY = 'TO_USERKEY'
 }
 
-/**
- * @deprecated
- */
```

Comment:
Isn't that still deprecated? It is used for `/custodial/*` endpoints which we do not want to use going forward, right?